Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
```

Comment:
I agree we don't need a "will not support" icon for now. I think "not supported" plus a link to an issue is sufficient (where the issue might explain why it is not supported now or that there are no plans to support it)

I also believe we have (and will have more) WCF features that we find do not work on some platform because our dependent packages don't support it.  Examples: NegotiateStream in NET Native, or using the X509Store in certain Linux distros.  I think having a single "not supported" icon is sufficient to cover this case, as long as it has a link to explain why it is not supported.